<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputWidget</class>
 <widget class="QWidget" name="InputWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>143</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AR PL UKai CN</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="key_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1!</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2@</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4$</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_8">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6^</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_7">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7&amp;&amp;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_0">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_Backspace">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string> back</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/icon/left/image/left.ico</normaloff>:/icon/left/image/left.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="key_W">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>W</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_T">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>T</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_Y">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_E">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>E</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_I">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_U">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_Q">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_P">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>P</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_R">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>R</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_O">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>O</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_S">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_F">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_G">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>G</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_D">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_A">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_K">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_J">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_H">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>H</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_L">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>L</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_X">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_V">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>V</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_B">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>B</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_C">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_Z">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_N">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>N</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_equal">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>=+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_caps">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>CAPS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_enter">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_M">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_comma">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>,&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_dot">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>.&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_dash">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>-_</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_quotation">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>'&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_semicolon">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>;:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_slash">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>/?</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_backslash">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>\|</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_lbracket">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>[{</string>
   </property>
  </widget>
  <widget class="QPushButton" name="key_rbracket">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>31</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>]}</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
